source/draw/RasterMonochrome.ooc

```diff
@@ -25,7 +25,7 @@ import Image, FloatImage
 import Color
 import Canvas, RasterCanvas
 
-MonochromeRasterCanvas: class extends RasterCanvas {
+MonochromeRasterCanvas: class extends PackedRasterCanvas {
```

Comment:
Yes maybe `RasterMonochromeCanvas` is better. Can you fix this in a new PR after this?